### ISSUE TITLE
Train labelled/unlabeled split for semi-supervised setup  

### ISSUE THREAD

**Issue by muqeemmm** [HUANGLIZI/LViT#49]
I have been following your work and I have been unable to find the labeled/unlabeled split for QaTa-COV19 and MosMedData+ used in the semi-supervised setup. To replicate the results proposed in the paper what splitting strategy should I use? If there is a split already available, please do let me know.

**Comment by HUANGLIZI** [comment]
Please use train_dataset, test_dataset = torch.utils.data.random_split(full_dataset, [train_size, test_size]) to split the dataset, and the random seed is the same as others.

**Comment by muqeemmm** [comment]
> Please use train_dataset, test_dataset = torch.utils.data.random_split(full_dataset, [train_size, test_size]) to split the dataset, and the random seed is the same as others.

Thank you. I am closing the issue.

**Comment by muqeemmm** [comment]
Thank you. I am closing the issue.
